basyx/aas/adapter/http.py

```diff
@@ -875,6 +884,48 @@ def delete_submodel_submodel_elements_id_short_path(self, request: Request, url_
         self._namespace_submodel_element_op(parent, parent.remove_referable, id_short_path[-1])
         return response_t()
 
+    def get_submodel_submodel_element_attachment(self, request: Request, url_args: Dict, **_kwargs) \
+            -> Response:
+        submodel_element = self._get_submodel_submodel_elements_id_short_path(url_args)
+        if isinstance(submodel_element, model.Blob):
+            return Response(submodel_element.value, content_type=submodel_element.content_type)
+        elif isinstance(submodel_element, model.File):
+            try:
+                with open(submodel_element.value, 'rb') as file:
+                    file_content = file.read()
+                return Response(file_content, content_type=submodel_element.content_type)
```

Comment:
This allows a client to read any file on the filesystem where the server is running. IIRC `File` objects can also reference non-local files, which we're unable to open. Because of these issues, I think we should only implement the attachment routes for `Blob` objects to now.

Reply:
Ok 👍